src/app/container/container.component.html

```diff
@@ -116,19 +124,34 @@ <h3>
         </form>
       </tab>
       <tab heading="Versions" *ngIf="validVersions">
-        <app-versions-container [versions]="tool?.tags" id="versionsTab" [verifiedSource]="tool?.verifiedSources">
+        <app-versions-container [versions]="tool?.tags" id="versionsTab" [verifiedSource]="tool?.verifiedSources" [selectedVersion]="selectedVersion">
```

Comment:
This brings up an interesting question. How do we want verification to look if some versions are verified and some are not?

Reply:
Currently we have the following:
* Checkmark beside entry name to show that at least one version is verified
* In versions tab, column to show which versions are verified
* In side bar for public viewing, we have a box with the verified versions